variables.tf

```diff
@@ -5,12 +5,27 @@ variable "resource_group_name" {
 }
 
 # Custom security rules
-# [priority, direction, access, protocol, source_port_range, destination_port_range, description]"
+# [{name, priority, destination_port_range}]"
```

Comment:
The comment also says `# All the fields are required.` but analysing the code you realise that only `name, priority, destination_port_range` are really required because the other attributes have defaults assigned. For instance:

`protocol = lookup(var.custom_rules[count.index], "protocol", "*")`
`access = lookup(var.custom_rules[count.index], "access", "Allow")`